judgels-backends/jophiel/jophiel-api/src/main/java/judgels/jophiel/api/session/SessionService.java

```diff
@@ -21,4 +21,9 @@
     @POST
     @Path("/logout")
     void logOut(@HeaderParam(AUTHORIZATION) AuthHeader authHeader);
+
+    @POST
+    @Path("/batch-logout")
+    @Consumes(APPLICATION_JSON)
+    void batchLogout(@HeaderParam(AUTHORIZATION) AuthHeader authHeader, BatchLogoutData data);
```

Comment:
```suggestion
    void batchLogOut(@HeaderParam(AUTHORIZATION) AuthHeader authHeader, BatchLogoutData data);
```

(resource methods are verbs)